chaoslib/litmus/disk_fill/disk_fill_by_litmus.yml

```diff
@@ -61,17 +61,20 @@
 
     - name: Patch the engine UID
       template:
-        src:  /chaoslib/litmus/disk_fill/disk_fill_ds.j2
-        dest: /chaoslib/litmus/disk_fill/disk_fill_ds.yml
-      
-    - name: Create the DaemonSet for Disk-Fill
+        src:  /chaoslib/litmus/disk_fill/disk_fill_job.j2
+        dest: /chaoslib/litmus/disk_fill/disk_fill_job.yml
+      vars:
+        chaosUID: "{{ chaos_uid }}"
+        app_node: "{{ app_node }}"
```

Comment:
If the name of both variables are the same why are you passing this? Why can you directly use the `app_node`  variable there without passing it?